docs/topics/settings.rst

```diff
@@ -604,6 +604,21 @@ If you want to disable it set to 0.
 
     This feature needs Twisted >= 11.1.
 
+.. setting:: DOWNLOAD_ALLOW_DATALOSS
+
+DOWNLOAD_ALLOW_DATALOSS
```

Comment:
The name is tricky :) Arguably we allow data loss when we don't pass partial content to the callback and drop the whole response instead. I don't have a better naming idea though.